cli/args/flags.rs

```diff
@@ -4996,8 +4997,22 @@ fn permission_args_parse(flags: &mut Flags, matches: &mut ArgMatches) {
   }
 
   if let Some(net_wl) = matches.remove_many::<String>("allow-net") {
-    let net_allowlist = flags_net::parse(net_wl.collect()).unwrap();
-    flags.permissions.allow_net = Some(net_allowlist);
+    match flags_net::parse(net_wl.collect()) {
+      Ok(net_allowlist) => {
+        flags.permissions.allow_net = Some(net_allowlist);
+      }
+      Err(e) => {
+        logger::init(Option::from(Level::Error));
+        let msg: String = e.to_string();
+        log!(
+          Level::Error,
+          "{}: {}",
+          colors::red_bold("error"),
+          msg.trim_start_matches("error: ")
+        );
+        std::process::exit(e.exit_code())
```

Comment:
This seems much too low level for this part of the code. Probably a result should be surfaced up instead.

Reply:
but this method it's void, and has been called in many methods, its needs many changes, I found that is a better solution that doesn't effect many methods, but if u want to return a `Result<(),Anyerror>` I will do it

Reply:
Thinking about it more, isn't there some way we could validate this with clap instead?

Reply:
@dsherret 
I tried clap, but it gave an error when I used `'?'`, so I converted it to `'Result<(),Anyerror>',` handled all necessary modifications, and used `downcast_ref `error as clap. 

I pushed changes You can check it.

Reply:
@dsherret 

i will try to use 
`-> clap::error::Result<()>`

Reply:
@dsherret 
done with return `clap::error::Result<()>`